tests/Unit/ControlSystem/Test_Initialization.cpp

```diff
@@ -87,16 +98,31 @@ SPECTRE_TEST_CASE("Unit.ControlSystem.Initialization",
       ActionTesting::get_databox_tag<component,
                                      control_system::Tags::TimescaleTuner>(
           runner, 0);
+  const auto& box_controller =
+      ActionTesting::get_databox_tag<component,
+                                     control_system::Tags::Controller<2>>(
+          runner, 0);
   const auto& box_controlsys_name =
       ActionTesting::get_databox_tag<component,
                                      control_system::Tags::ControlSystemName>(
           runner, 0);
 
+  // Check that things haven't been initialized
+  CHECK(box_averager != averager);
+  CHECK(box_tuner != tuner);
+  CHECK(box_controlsys_name != controlsys_name);
+  // We don't check the controller now because one of its members is
+  // inititalized with signaling_NaN() so comparing now would throw an FPE.
+
+  // Now initialize everything
+  runner.force_next_action_to_be<
+      component, Initialization::Actions::InitializeControlSystem<
+                     MockMetavars, mock_control_sys>>(0);
```

Comment:
This is already the next action, isn't it?  Is there something I'm missing going on here?

Reply:
This may have something to do with the previous comment about initialization, I'm not sure. But I tried `ActionTesting::next_action` like you are suggesting and that gave an error. Then I tried to just see how many actions were left with `ActionTesting::number_of_queued_simple_actions` and it was 0. Even though I hadn't ran any actions. I was very confused by this.
So this was the only way I could get the action to run.

Reply:
It's not a simple action, it's an iterable action, so that part is expected.

Testing, it seems to work without this.  What problem are you seeing?

Reply:
I think I was just using the ActionTesting framework incorrectly. I got it to work now.